tests/utils/replicaset/replicaset.go

```diff
@@ -38,3 +38,19 @@ func RedefineWithReplicaNumber(replicaSet *v1.ReplicaSet, replicasNumber int32)
 
 	return replicaSet
 }
+
+func RedefineWithVolume(replicaSet *v1.ReplicaSet, name string) *v1.ReplicaSet {
```

Comment:
I don't think you need to return the pointer here as it's already pass-by-reference.  Just modify the pointer in place.